src/js/components/why-eventplog/WhyEventplog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledWhyEventplog = styled.div`
+  display: flex;
+  justify-content:center;
+
+    .why-eventplog-content {
+      max-width: 700px;
+      line-height: 30px;
+      margin: 2rem;
+    }
+
+    .topic {
+      text-align:center;
+    }
+`
+
+const WhyEventplog = () => {
+  return(
+    <StyledWhyEventplog className="app-container">
+      <div className="why-eventplog-content">
+        <h5 className="topic">Why EventPlog </h5>
+        <p>
+          EventPlog is built by community organizers participating in more than five 
+          different communities. Some have organized event while others has volunteered in 
+          helping out with events or meetups planning. We know the stress and the challenges it 
+          entails to plan a successful event.</p>
```

Comment:
Great job on this page! Building on what you've written, here's what I'd say

=============

EventPlog is built by community organizers who are active administrators of more than five 
different communities. By the end of the first half of 2018, we directly planned 26 events (at least one event every week). We main a [widely-read blog](techmeetupsng.com) that celebrate tech events, people and communities.

**Our Problem**
We use Splashthat and Eventbrite for RSVPs, twitter, emails and Facebook to reach out to guests through social media. It doesn't take long to find a few issues:
1. We couldn't get guests to be as involved in the process as we would like.
2. It wasn't very easy getting sponsorships from local firms.
3. Our `not-so-experienced` planning teams of community volunteers needed a bit longer to sync into productive planning.
4. Accountability throughout the entire planning process takes A LOT of effort to maintain.

**Our solution**
Eventplog focuses on the solving these problems for guests, sponsors and event planners. It's not very uncommon to see on person having all three roles. 

For event planners, we are making it easy to accountably plan events, improve discoverability, get sponsorships and carry guests along as much as you would like.

For guests, we are making it easier to stay close to events you're interested in, give your input on these events and document your learnings from these events.

For sponsors, we are making it easier to reliably sponsor events that are genuine, have measurable impact and are very accountable.

**The big picture**

Education systems around the world are getting more democratic. Communities are becoming important centers for learning and bridging skill gaps.

The better we are able to get the people who play all three roles to work together towards building impactful events, the faster a community can cheaply up-skill its workforce.

To us, working together remains key.

==============